composables/useTransaction.ts

```diff
@@ -82,21 +91,28 @@ export const resolveSuccessMessage = (
 const useExecuteTransaction = (options: TransactionOptions) => {
   const { accountId } = useAuth()
   const error = ref(false)
+
   const {
     howAboutToExecute,
     isLoading,
     status,
     initTransactionLoader,
     isError,
-  } = useMetaTransaction()
+  } = execByVm({
+    SUB: () => useMetaTransaction(),
+    EVM: () => useEvmMetaTransaction(),
```

Comment:
im getting

Property 'onTxError' is missing in type...

Reply:
fixed